test/helpers/utils.js

```diff
@@ -40,18 +40,27 @@ exports.checkOutput = function (f, argv, cb) {
 
     f()
   } else {
-    result = f()
+    try {
+      result = f()
+    } finally {
+      reset()
```

Comment:
Seeing as how this code doesn't change much and the operation is idempotent, I'm fine leaving it this way.